p2p/simulations/p2ptest.go

```diff
@@ -75,14 +87,14 @@ func main() {
 		//slice29, _ := hexutil.HexToBytes(node29)
 		//fmt.Println(slice29)
 		nodeID29 := common.HexToAddress(node29)
-		addr29, _ := net.ResolveUDPAddr("udp4", "182.87.223.29:39009")
+		addr29, _ := net.ResolveUDPAddr("udp4", "182.87.223.29:39008")
```

Comment:
where is 182.87.223.29?
and nice to define the port 39008 as constant or to be configuable

Reply:
Should read from config